### PR TITLE
Random logo

### PR DESCRIPTION
**Pull request by NacymeG**

### CODE REVIEW

**Review by Caroline-CD — DISMISSED** [review]
OK for me

**Review by John-athan14 — APPROVED** [review]

File: src/App.jsx [hunk 1]
```diff
@@ -10,6 +11,7 @@ function App() {
       <Header />
       <Switch>
         <Route exact path="/" component={Home} />
+        <Route exact path="/Cards/" component={CardDescription} />
```
Comment (Hacene010) [inline src/App.jsx:14]: Why did you add the last "/" in your path ?. 
You don't need to specify exact path for all your routes (just one exact path for your home is enough)

File: src/Components/Header/Navigation/Random/Random.jsx [hunk 2]
```diff
@@ -1,13 +1,19 @@
...
 const Random = () => {
   return (
-    <li>
-      <img
-        className="random logo-header"
-        src="/assets/img/random.png"
-        alt="random"
-      />
-    </li>
+    <Link to="/Cards/">
```
Comment (Hacene010) [inline src/Components/Header/Navigation/Random/Random.jsx:6]: same here about the "/" at the end of Cards

File: src/Containers/CardDescription.jsx [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+          alt="A magic card"
+        />
+        <div className="CardAttribute">
+          <p className="CardName">{card.name}</p>
+          <p className="CardType">{card.type_line}</p>
+          <p className="CardOracle">{card.oracle_text}</p>
+          {card.flavor_text ? (
+            <p className="CardFlavour"> {card.flavor_text}</p>
+          ) : (
+            ''
```
Comment (Hacene010) [inline src/Containers/CardDescription.jsx:150]: Use null instead of the empty string ??

**Review by Hacene010 — APPROVED** [review]
You can do better than that.
check the comments for the next changes ; )
